app/routers/categories.py

```diff
@@ -43,22 +45,57 @@ def get_categories(request: Request,
                                    f"unallowed params.")
 
 
+@router.get("/event/edit")
+def your_categories(request: Request,
+                    db_session: Session = Depends(get_db)) -> List[Category]:
+    """
+    will add all user categories to dropdown list
+    """
+    user_id = 1
+    categories_list = get_user_categories(db_session, user_id)
+    return templates.TemplateResponse("event/eventedit.html", {
+        "request": request,
+        "categories_list": categories_list,
+    })
+
+
+@router.get("/categories")
+def category_color_insert(request: Request) -> _TemplateResponse:
+    return templates.TemplateResponse("categories.html", {
+        "request": request
+    })
+
+
 # TODO(issue#29): get current user_id from session
-@router.post("/")
-async def set_category(category: CategoryModel,
-                       db_sess: Session = Depends(get_db)) -> Dict[str, Any]:
+@router.post("/categories")
+async def set_category(request: Request,
+                       category_name: str = Form(None),
```

Comment:
You should send from FE a post request with body which includes a categoryModel == a json (dict).
Doesn't it work?